cajita/src/Cajita_Parallel.hpp

```diff
@@ -17,6 +17,10 @@
 
 #include <string>
 
+#ifdef KOKKOS_ENABLE_OPENMPTARGET // FIXME_OPENMPTARGET
+#error Not supported when Kokkos::Experimental::OpenMPTarget backend enabled.
+#endif
```

Comment:
We could also catch that at configure time

Reply:
Done